docs/csharp/language-reference/operators/addition-operator.md

```diff
@@ -17,13 +17,13 @@ ms.assetid: 93e56486-bb42-43c1-bd43-60af11e64e67
 ---
 # + and += operators (C# reference)
 
-The `+` operator is supported by the built-in numeric types, [string](../keywords/string.md) type, and [delegate](../keywords/delegate.md) types.
+The `+` and `+=` operators are supported by the built-in [integral](../builtin-types/integral-numeric-types.md) and [floating-point](../builtin-types/floating-point-numeric-types.md) numeric types, the [string](../builtin-types/reference-types.md#the-string-type) type, and [delegate](../builtin-types/reference-types.md#the-delegate-type) types.
```

Comment:
I think it should be singular here, as with `the string type`

```suggestion
The `+` and `+=` operators are supported by the built-in [integral](../builtin-types/integral-numeric-types.md) and [floating-point](../builtin-types/floating-point-numeric-types.md) numeric types, the [string](../builtin-types/reference-types.md#the-string-type) type, and [delegate](../builtin-types/reference-types.md#the-delegate-type) type.
```

Reply:
There are a lot of different delegate types; for example, `Action<int>` and `Func<int, double>` are distinct types. Instances of different delegate types cannot be combined.